scripts/overrides/file_csv.rb

```diff
@@ -31,9 +34,12 @@ def transform_es(old_case_docs, new_case_docs)
     end
 
     def read_csv(file_location, encoding="utf-8")
-        CSV.read(file_location, **{
-          encoding: encoding,
-          headers: true
+      CSV.read(file_location, **{
+        encoding: encoding,
+        headers: true,
+        return_headers: true,
+        :header_converters=> lambda {|f| f.strip},
+      :converters=> lambda {|f| f ? f.strip : nil}
         })
```

Comment:
Same style questions as Datura PR:
- Firstly, the second converters line's indentation doesn't match the other options lines. And I think the closing `})` should be unindented to match the indentation of the `CSV.read(...` line.
- Also, can these be written as `header_converters: lambda ...` and `converters: lambda ...` to match the other options syntax? Options keys are symbols so we shouldn't need the older, more verbose hash rocket syntax and the Ruby style guide dictates not mixing syntax styles as well.